OTAnalytics/plugin_ui/customtkinter_gui/frame_flows.py

```diff
@@ -69,6 +66,21 @@ def _place_widgets(self) -> None:
     def action_buttons(self) -> list[CTkButton]:
         return self._action_buttons
 
+    def set_selection(self, flow_ids: list[str]) -> None:
+        pass
```

Comment:
In `frame_sections.py` a `NotImplementedError` is raised. Should it be the same behaviour? Or should we remove the method completely?

Reply:
Yes, it is not being used so we can remove method completely.